utility/src/main/java/org/oppia/util/logging/ExceptionsHelper.kt

```diff
@@ -0,0 +1,47 @@
+package org.oppia.util.logging
+
+import org.oppia.app.model.ExceptionLog
+
+/** Helper class for conversion of exceptionLog proto object to an Exception. */
+class ExceptionsHelper {
+
+  /** Converts the [exceptionLog] into an appropriate exception. */
+  fun convertExceptionLogToException(exceptionLog: ExceptionLog): Exception {
+    val exceptionMessage = if (exceptionLog.message != "") {
+      exceptionLog.message
+    } else {
+      null
+    }
+    val exceptionCause: Throwable? =
+      if (exceptionLog.cause.message == "" && exceptionLog.cause.stacktraceElementCount == 0) {
+        null
+      } else if (
+        exceptionLog.cause.message == "" && exceptionLog.cause.stacktraceElementCount != 0
+      ) {
+        Throwable()
+      } else {
+        Throwable(exceptionLog.cause.message)
+      }
+    exceptionCause?.let {
+      it.stackTrace = createErrorStackTrace(exceptionLog.cause)
+    }
```

Comment:
```suggestion
    val exceptionCause: Throwable? =
      if (exceptionLog.hasCause()) convertExceptionLogToException(exceptionLog.cause)
      else null
```

This seems simplifiable.

Reply:
Yes, done.